src/database/database.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <iostream>
+#include <cstdint>
+#include <vector>
+
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+#include <mongocxx/stdx.hpp>
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+
+using bsoncxx::builder::basic::kvp;
```

Comment:
Be careful with `usings` in headers, they are propagated everywhere where the header is included. Consider whether they are worth.